IFTPipeline.jl/src/preprocess.jl

```diff
@@ -296,5 +296,194 @@ function preprocess_single(; truecolor::T, falsecolor::T, landmask::T, landmask_
     @info "Writing segmented floes to $output"
     save_labeled_img(labeled_floes_cast, output)
 
+    return nothing
+end
+
+"""
+    preprocess_tiling_single(; truecolor::T, falsecolor::T, landmask::T, landmask_dilated::T, output::T) where {T<:AbstractString}
+
+Preprocess and segment floes in a single view. Save the segmented floes to `segmented` and the labeled floes to `labeled`.
+
+# Arguments
+- `truecolor`: path to truecolor image
+- `falsecolor`: path to falsecolor image
+- `landmask`: path to landmask image
+- `landmask_dilated`: path to dilated landmask image
+- `segmented`: path to segmented output file 
+- `labeled`: path to labeled output file 
+"""
+function preprocess_tiling_single(
+    ; 
+    truecolor::T, 
+    falsecolor::T, 
+    landmask_dilated::T, 
+    segmented::T,
+    labeled::T,
+
+    # Tiling parameters
+    tile_rblocks=8,
+    tile_cblocks=8,
```

Comment:
I think @danielmwatkins also wanted some control for tiling by tile size, so another version of this function might be needed for such use case. See [`get_tiles`](https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/a7065e8b987f6349f1c55213deb823cb9b23086d/src/tilingutils.jl#L198).